crates/agent/src/lib.rs

```diff
@@ -17,9 +17,12 @@ use std::{
     },
     time::Duration,
 };
-use tokio::sync::{
-    mpsc::{self, UnboundedSender},
-    Mutex,
+use tokio::{
+    sync::{
+        mpsc::{self, UnboundedSender},
+        Mutex,
```

Comment:
nit: is Mutex still used?

Reply:
[Yup!](https://github.com/estuary/flow/pull/802/files#diff-c89117f7bd70bccf98ba0d48e2ed19e740a8c59c9d1b33cfe33fbc01cd8ebd7eL211)